README.md

```diff
@@ -131,7 +119,7 @@ Description=Foreman Cockpit Web Service
 
 [Service]
 Environment=XDG_CONFIG_DIRS=/etc/foreman-cockpit/
-ExecStart=/usr/libexec/cockpit-ws --no-tls --address 127.0.0.1 --port 9999
+ExecStart=/usr/libexec/cockpit-ws --no-tls --address 127.0.0.1 --port 19090
```

Comment:
Need to pick a port which is unused in SELinux base policy otherwise we'd need to detach it from its current type and attach it back to websm port (which is the one used by cockpit - 9999).

Reply:
This implies we also need to update the puppet module -- https://github.com/theforeman/puppet-foreman/blob/master/manifests/plugin/remote_execution/cockpit.pp#L9

Reply:
Yeah, let's do it (sorry was on PTO).